tempo_project/project.json5

```diff
@@ -0,0 +1,358 @@
+{
+  project_id: 'tempo_ldv_profiles_2022',
+  description: 'TEMPO light-duty vehicle charging profiles created in 2021/2022 and published in 2023. https://www.nrel.gov/docs/fy23osti/83916.pdf',
+  name: 'TEMPO LDV Profiles 2022',
+  datasets: [
+    {
+      dataset_id: 'tempo_conus_2022',
+      dataset_type: 'modeled',
+      version: '1.0.0',
+      required_dimensions: {
+      },
+    },
+    {
+      dataset_id: 'tempo_conus_2022_mapped',
+      dataset_type: 'modeled',
+      version: '1.0.0',
+      required_dimensions: {
+      },
+    },
+  ],
+  dimensions: {
+    base_dimensions: [
+      {
+        class: 'County',
+        description: 'US counties for L48, Census 2020 vintage, no territories',
+        file: 'dimensions/counties.csv',
+        module: 'dsgrid.dimension.standard',
+        name: 'US Counties 2020 L48',
+        display_name: 'County',
+        type: 'geography',
+      },
+      {
+        class: 'Sector',
+        type: 'sector',
+        name: 'Transportation Sector',
+        display_name: 'Transportation',
+        file: 'dataset/dimensions/sector.csv',
+        description: 'Transportation Sector',
+      },
+      {
+        class: 'Subsector',
+        type: 'subsector',
+        name: "TEMPO Bin",
+        display_name: "Household and Vehicle Type",
+        file: "dataset/dimensions/bin.csv",
+        description: "TEMPO demographic/geographic/vehicle bins. This is a cross-join of the following supplemental dimensions: * Composition (Single Driver, Some Drivers Smaller, Some Drivers Larger) * Income (Low Income, Middle Income, High Income) * Urbanity (Rural, Small Town, Suburban, Second City, Urban) * Vehicle Size Class (Compact, Midsize, SUV, Pickup) * Tech (PHEV25/PHEV50/BEV100/BEV300) ",
+      },
+      {
+        class: 'EnergyEndUse',
+        description: 'dsgrid CONUS 2022 End Uses (all fuel types)',
+        file: 'dimensions/enduses.csv',
+        module: 'dsgrid.dimension.standard',
+        name: 'CONUS-2022-Detailed-End-Uses',
+        display_name: 'End Use',
+        type: 'metric',
+      },
+      {
+        class: "ModelYear",
+        type: "model_year",
+        name: "TEMPO Model Year",
+        display_name: "TEMPO Model Years",
+        file: "dataset/dimensions/year.csv",
+        description: "TEMPO modeling years for CONUS 2022 project, 2018-2050",
+      },
```

Comment:
Dan votes for expanding this to all model years.

Matteo probably votes for starting at 2024, and folks are largely in agreement.

Reply:
To do these, we'd make a new project year file and an accompanying mapping.

Reply:
So, tempo_project/dataset/dimensions/year.csv should start at 2024
and 
there should be a  dimension_mappings/year_to_year.csv (or is it model_year_to_model_year.csv)?
from_id,to_id
2018,
2020,
2022,
2024,2024
2026,2026

and an entry in tempo_project/dataset/dimension_mappings.json5
but I'm not sure of the dimension_type or mapping_type.

Reply:
About expanding to all years, I'm ok either way (including or excluding the interpolated years). 
It has a little bit of additional value, so I would say yes only if it's not any additional work.
If it's easier to ignore/exclude, I would vote that way.